src/utils/toggleScroll.js

```diff
@@ -0,0 +1,52 @@
+const keys = [37, 38, 39, 40, 32, 9];
+
+const preventDefaultScroll = (e, el) => {
+    if (
+        !el ||
+        !e.path.includes(el) ||
+        (el.scrollTop === el.scrollHeight - el.offsetHeight && e.deltaY > 0) ||
+        (el.scrollTop === 0 && e.deltaY < 0)
+    ) {
+        e.preventDefault();
+        return false;
+    }
+};
+const preventDefaultKeys = (e) => {
+    if (keys.indexOf(e.keyCode) !== -1) {
```

Comment:
The `keyCode` is deprecated, you should use `code`,
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
The code is not supported in Internet Explorer, so this discussion is resolved